modules/basegl/processors/volumeslicegl.cpp

```diff
@@ -736,6 +737,31 @@ void VolumeSliceGL::rotationModeChange() {
     }
 }
 
+void VolumeSliceGL::updateFromWorldPosition() {
+
+    if (!inport_.hasData() || updating_) return;
+
+    if (inport_.hasData()) {
+        util::KeepTrueWhileInScope guard(&updating_);
+
+        const mat4 worldToTexture(
+            inport_.getData()->getCoordinateTransformer().getWorldToTextureMatrix());
+        const vec3 texturePos = vec3(worldToTexture * vec4(worldPosition_.get(), 1.0f));
+
+        const mat4 worldToIndex(
+            inport_.getData()->getCoordinateTransformer().getWorldToIndexMatrix());
+        const ivec3 indexPos(ivec3(worldToIndex * vec4(worldPosition_.get(), 1.0f)) + ivec3(1));
+
+        NetworkLock lock(this);
+        sliceX_.set(indexPos.x);
+        sliceY_.set(indexPos.y);
+        sliceZ_.set(indexPos.z);
+
+        planePosition_.set(texturePos);
+        invalidateMesh();
+    }
```

Comment:
Nice, Since this on updates both slize* and placePosistion I think it would be more constant if sliceChange and positionChange also updated worldPosistion instead of renderPositionIndicator as it is now...